filetracker/scripts/migrate.py

```diff
@@ -0,0 +1,98 @@
+"""Script to upload a file tree to a remote filetracker server.
+
+The intention for this script is to support migration to new filetracker
+servers that change the format of disk storage. It should be used with
+redirect functionality of the new server.
```

Comment:
What is redirect functionality of the new server? Should I know anything about it? Do I need to enable it? The sentence suggests that I need to do something special with the new server to use this script. Is it really the case? If not, then I think this sentence is not needed.